.swiftlint.yml

```
@@ -0,0 +1,573 @@
+    - type_name # new
+    
+    # 불필요한 break 문을 사용하지 마십시오. https://realm.github.io/SwiftLint/unneeded_break_in_switch.html
+    - unneeded_break_in_switch # new
+    
+    # 클로저 인수를 선언 할 때는 괄호가 필요하지 않습니다. https://realm.github.io/SwiftLint/unneeded_parentheses_in_closure_argument.html
+    - unneeded_parentheses_in_closure_argument # new
+    
+    # 사용하지 않는 클로저내 캡쳐는 하지 않습니다. https://realm.github.io/SwiftLint/unused_capture_list.html
+    - unused_capture_list # new
+    
+    # 사용하지않는 클로저 파라미터는 _로 대체합니다. https://realm.github.io/SwiftLint/unused_closure_parameter.html
+    - unused_closure_parameter
+    
+    # 사용하지 않는 제어문 이름은 제거해야합니다. https://realm.github.io/SwiftLint/unused_control_flow_label.html
+    - unused_control_flow_label
+    
+    # index를 사용하지 않으면 enumrated를 사용하지 않습니다. https://realm.github.io/SwiftLint/unused_enumerated.html
+    - unused_enumerated # new
+
+    # 사용하지 않는 import는 제거 합니다. https://realm.github.io/SwiftLint/unused_import.html
+    - unused_import # new
+    
+    # 사용하지 않는 옵셔널 바인딩은 제거 합니다. https://realm.github.io/SwiftLint/unused_optional_binding.html
+    - unused_optional_binding # new
+    
+    # set를 사용하지 않으면 제거 합니다. https://realm.github.io/SwiftLint/unused_setter_value.html
+    - unused_setter_value # new
+    
+    # @IBInspectable은 변수에만 적용되어야하며 명시적인 유형을 가지며 지원되는 유형이어야합니다. https://realm.github.io/SwiftLint/valid_ibinspectable.html
+    - valid_ibinspectable # new
+    
+    # 함수 매개변수들이 여러개일경우 세로로 같은줄에 있어야 합니다. https://realm.github.io/SwiftLint/vertical_parameter_alignment.html
+    - vertical_parameter_alignment # new
+    
+    # 함수를 사용할때 인자들이 여러개일경우 세로로 같은줄에 있어야 합니다. https://realm.github.io/SwiftLint/vertical_parameter_alignment_on_call.html
+    - vertical_parameter_alignment_on_call
+    
+    # 수직으로는 여백이 2줄이상 생기면 안됩니다. https://realm.github.io/SwiftLint/vertical_whitespace.html
+    - vertical_whitespace # new
+
+    # 닫는 괄호 앞에는 공백이 있으면 안됩니다. https://realm.github.io/SwiftLint/vertical_whitespace_closing_braces.html
+    - vertical_whitespace_closing_braces # new
+    
+    # 여는 괄호 다음에는 공백이 있으면 안됩니다. https://realm.github.io/SwiftLint/vertical_whitespace_opening_braces.html
+    - vertical_whitespace_opening_braces # new
+    
+    #  -> () 보단 -> Void 를 선호합니다.
+    - void_return
+    
+    # delegate는 weak로 합니다. https://realm.github.io/SwiftLint/weak_delegate.html
+    - weak_delegate # new
+    
+    # xctfail에는 메세지가 있어야 합니다. https://realm.github.io/SwiftLint/xctfail_message.html
+    - xctfail_message # new
+    
+    # 비교 할대 변수는 왼쪽 상수는 오른쪽에 있어야 합니다. https://realm.github.io/SwiftLint/yoda_condition.html
+    - yoda_condition
+    
+# Options
+type_body_length:
+  warning: 200 # warning
+  error: 400 # error
+file_length:
+  warning: 300
+  error: 600
+function_body_length:
+  warning: 20
+  error: 40
+  
+reporter: "xcode"
+
+excluded:
+    - Carthage
+    - WeatherMood/*/AppDelegate.swift
+    - WeatherMood/*//SceneDelegate.swift
```

```suggestion
    - WeatherMood/*/SceneDelegate.swift
```
`/`가 하나 더 들어간 거 같은데 아닌가요?

---

네 맞아요 감사합니다